WcaOnRails/app/views/competitions/_competition_form.html.erb

```diff
@@ -28,14 +28,17 @@
       <% disable_form = !can_edit_competition %>
 
       <fieldset <%= disable_form && "disabled" %>>
+        <% edit_cellName_id = !(@competition_admin_view || (@competition.name.length > 32)) %>
```

Comment:
Are you sure the second pair of brackets is needed?
Also, please reference `Competitions::MAX_CELL_NAME_LENGTH` instead of magically inserting the literal `32` out of nowhere.

Reply:
About the first question: Yes? If I do `edit_cellName_id = !@competition_admin_view || (@competition.name.length > 32)` then the name/nickname is not possible to edit from the organiser view, despite the name being over 32 chars.
(second half done)